core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -6,16 +6,30 @@
       </h4>
     </div>
     <a href="https://plus.google.com/109898456505810251700/about" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/gplus.png')]>" alt="Google+">
+      <i class="oppia-share-btn-fa fa-google-plus-square"></i>
     </a>
     <a href="https://www.youtube.com/channel/UC5c1G7BNDCfv1rczcBp9FPw" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/youtube.png')]>" alt="YouTube">
+      <i class="oppia-share-btn-fa fa-youtube-play"></i>
     </a>
     <a href="https://www.facebook.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/fb.png')]>" alt="Facebook">
+      <i class="oppia-share-btn-fa fa-facebook"></i>
     </a>
     <a href="https://twitter.com/oppiaorg" target="_blank">
-      <img ng-src="<[getStaticImageUrl('/social/twitter.png')]>" alt="Twitter">
+      <i class="oppia-share-btn-fa fa-twitter"></i>
     </a>
   </div>
 </script>
+
+<style>
+  i.oppia-share-btn-fa {
+   	display: inline-block;
+   	font: normal normal normal 14px/1 FontAwesome;
+   	font-size: 23px;
+   	padding-right: 4px;
+   	position: relative;
+   	text-rendering: auto;
+   	top: -10px;
+   	-moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+  }
+</style>
```

Comment:
I didn't run presubmit checks. Indentation in my sublime text looks so.. 
Don't know why commit shows wrong indent? 
Cause of the tabs thing? 
![indent](https://cloud.githubusercontent.com/assets/24438869/23847175/26ea90a8-07f7-11e7-9ef1-fbf1a0a77939.jpg)